bucket/tftputil-gui.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.4.5",
+    "description": "TFTP server and client written in .NET framework.",
+    "homepage": "https://sourceforge.net/projects/tftputil/",
+    "license": "0BSD",
+    "url": "https://dl.sourceforge.net/project/tftputil/TFTPUtil/TFTPUtil%20Version%201.4.5/TFTPUtil_GUI_Version_1.4.5_Binary_Installer.exe#/dl.7z",
+    "hash": "sha1:4f7ecfd0abdd2f7c48438ce64e22981fba6daff4",
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\src\", \"$dir\\uninstall.exe\" -Force -Recurse",
+    "bin": [
+        "TFTPUtil GUI.exe",
+        "TFTPUtil Client GUI.exe"
+    ],
+    "shortcuts": [
+        [
+            "TFTPUtil GUI.exe",
+            "TFTPUtil GUI"
+        ],
+        [
+            "TFTPUtil Client GUI.exe",
+            "TFTPUtil Client GUI"
+        ]
+    ],
+    "checkver": "TFTPUtil_GUI_Version_([\\d.]+)",
+    "autoupdate": {
+        "url": "https://dl.sourceforge.net/project/tftputil/TFTPUtil/TFTPUtil%20Version%20$version/TFTPUtil_GUI_Version_$version_Binary_Installer.exe#/dl.7z"
```

Comment:
Why did you select the slowest mirror, which is not working most of the time for wide variety of countries?

```suggestion
        "url": "https://downloads.sourceforge.net/project/tftputil/TFTPUtil/TFTPUtil%20Version%20$version/TFTPUtil_GUI_Version_$version_Binary_Installer.exe"
```

![image](https://user-images.githubusercontent.com/13260377/70853217-602ca180-1eab-11ea-8ffb-d9a1c8366b76.png)

Reply:
Didn't know that. Thanks for pointing that out.